bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeper.java

```diff
@@ -614,7 +615,19 @@ public void safeRun() {
         }
     }
 
+    boolean enableHealthCheck() {
+        if (metadataDriver instanceof ZKMetadataClientDriver) {
```

Comment:
why this "instanceof" ?

please add the method in the MetadataClientDriver class

Reply:
You are right